packages/webamp/js/components/FFTNullsoft.ts

```diff
@@ -0,0 +1,243 @@
+// The Web Audio API's FFT is bad, so this exists now!
+// Taken from https://github.com/WACUP/vis_classic/tree/master/FFTNullsoft
+
+export class FFT {
+    private m_samples_in: number;
```

Comment:
Can we use `camelCase` throughout instead of `snake_case` to keep consistent with the rest of Webamp?

Reply:
Sure!